loopd/config.go

```diff
@@ -26,17 +26,22 @@ type lndConfig struct {
 	TLSPath     string `long:"tlspath" description:"Path to lnd tls certificate"`
 }
 
+type loopServerConfig struct {
+	Host  string `long:"host" description:"Loop server address host:port"`
```

Comment:
missing punctuation?

Reply:
I was in doubt. Do we or don't we do punctuation? It seems we don't if the sentence is 'short enough' :grimacing:

Reply:
It doesn't really matter imo, but we could be consistent. Checked LND for inspiration and it mostly doesn't have punctuations unless it does :)

Reply:
I vote for consistently not using punctuation unless there's more than one sentence (which probably should be avoided anyway).

Reply:
Sounds like a rule. Applied.